CMakeLists.txt

```
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(PortSMF VERSION 1.0)
```

Probably not needed, but good to be explicit, IMO.
```suggestion
project(PortSMF VERSION 1.0 LANGUAGES CXX)
```